rh-tictactoe/src/app/components/Topbar/index.js

```diff
@@ -3,26 +3,24 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { NavLink } from 'react-router-dom';
 
-import { actionCreator } from '../../../../src/redux/topbar/actions.js';
+import { actionCreator } from '../../../../src/redux/game/actions.js';
 
 import Styles from './styles.scss';
 
-class Links extends React.Component {
-  render() {
-    return (
-      <nav className={Styles.topbar}>
-        <NavLink className={Styles.link} onClick={this.props.logOut} to="/login">
-          Log Out
-        </NavLink>
-        <NavLink className={Styles.link} to="/app/game">
-          TicTacToe
-        </NavLink>
-        <NavLink className={Styles.link} to="/app/stats">
-          Stats
-        </NavLink>
-      </nav>
-    );
-  }
+function Links(props) {
```

Comment:
when i mean function component is always in arrow form and destructing props values